internal/api/api.go

```diff
@@ -23,6 +23,18 @@ const (
 	writeTimeout = 10 * time.Second
 )
 
+//	@title		SubFlow API
+//	@version	1.0
+
+//	@license.name	GPLv3
+//	@license.url	https://www.gnu.org/licenses/gpl-3.0.html
+
+//	@host		localhost:8000
+//	@BasePath	/v1
+
+// @securityDefinitions.apiKey	ApiKeyAuth
+// @In							header
+// @Name						Authorization
```

Comment:
```suggestion
//	@title		SubFlow API
//	@version	1.0

//	@license.name	GPLv3
//	@license.url	https://www.gnu.org/licenses/gpl-3.0.html

//	@host		localhost:8000
//	@BasePath	/v1

//	@securityDefinitions.apiKey	ApiKeyAuth
//	@In				header
//	@Name				Authorization
```

These lines don't seem to be aligned (Is this because I use 8-character tab?). Maybe it's better to have them aligned.

Reply:
It's a problem TBD, because `swaggo/swag` forced to mixed use space and tab to indent.